README.md

```diff
@@ -53,11 +53,6 @@ resources of this module:
 The [Project Factory module][project-factory-module] can be used to
 provision a project with the necessary APIs enabled.
 
-## Contributing
```

Comment:
Could we clarify we are not accepting in the readme and also delete https://github.com/terraform-google-modules/terraform-google-sap/blob/master/CONTRIBUTING.md?